frontend/src/pages/accounts/profile.test.tsx

```diff
@@ -340,6 +338,7 @@ describe("The dashboard", () => {
       configurable: true,
     });
     render(<Profile />);
+    setMockMinViewportWidth(true);
```

Comment:
Question: This might not be needed since we are not setting `setMockMinViewportWidth(false)` before in this test?
```suggestion
```